modules/nw-networkpolicy-about.adoc

```diff
@@ -144,7 +144,7 @@ spec:
   - from:
     - namespaceSelector:
         matchLabels:
-          policy-group.network.openshift.io/ingress:""<1>
+          policy-group.network.openshift.io/ingress: ""<1>
```

Comment:
Understand if you want to leave this alone, since it was already like this, but if you want to format this the [prescribed way](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-asciidoc-callouts-in-the-yaml) while you are in here, it would be:

```suggestion
          policy-group.network.openshift.io/ingress: "" # <1>
```

Reply:
You can commit as is and it will copy the code only while highlighting. If we change, I think there will some issues while copying the code.